### PR TITLE
Rename data policies navigation to disaster recovery

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Issue: https://issues.redhat.com/browse/RHSTOR-5388

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
image

File: plugins/mco/console-extensions.json [hunk 1]
```diff
@@ -30,11 +30,11 @@
   {
     "type": "console.navigation/href",
     "properties": {
-      "id": "mco-data-policies",
+      "id": "mco-disaster-recovery",
       "perspective": "acm",
       "section": "mco-data-services",
-      "name": "%plugin__odf-multicluster-console~Data policies%",
-      "href": "/multicloud/data-services/data-policies"
+      "name": "%plugin__odf-multicluster-console~Disaster recovery%",
```
Comment (SanjalKatiyar) [inline plugins/mco/console-extensions.json:36]: there is a file `i18n-comments.ts` in odf, we need to create another one under "mco" and mark texts like `Disaster recovery`, `Data Services` etc there... else these will not get translated...

Comment (SanjalKatiyar) [reply plugins/mco/console-extensions.json:36]: it is a small thing, not blocking this PR... but something to keep notice of and fix later...

Comment (GowthamShanmugam) [reply plugins/mco/console-extensions.json:36]: ack

**Pull request merged**